packages/cactus-api-client/src/main/typescript/socketio-api-client.ts

```diff
@@ -375,7 +375,7 @@ export class SocketIOApiClient implements ISocketApiClient<SocketLedgerEvent> {
 
       return this.monitorSubject.pipe(
         finalize(() => {
-          if (this.monitorSubject && !this.monitorSubject.observed) {
+          if (this.monitorSubject && !this.monitorSubject.observers) {
```

Comment:
@aldousalvarez @petermetz `this.monitorSubject.observers` is always truthy (it's an array), `stopMonitor` is never emitted, causing #2191. What is the reason for this change?

Reply:
@outSH  Apologies, while doing the fix on upgrade migration of typescript to nodeNext the path of the imports changed. Based on the suggested fix observers must be used thus the issue. Upon checking, the observed method returns the observers.length and I've already implemented it to <**packages/cactus-api-client/src/main/typescript/socketio-api-client.ts**> which is now passing see the latest [commit](https://github.com/hyperledger/cactus/pull/2179/commits/7f6493e6a866b98a51b72664b5658e7fa867a42d).